TurkishFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@ fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfi
 !
 ||paraanaliz.com/wp-content/uploads/*/gedik_gif_banner_youtube.gif
 odatv4.com###menu-icon ~ div[style^="margin-top: "]
+spyhackerz.org##a[href="https://xleet.to/?ref=spy_banner"]
```

Comment:
Just `||i.ibb.co/*/ezgif-com-gif-maker-`

Reply:
What is the difference between them?

Reply:
In this case, it's better to block this gif with size 1.1 Mb.

Reply:
There are two types gif I guess. One is your rule, two is: `i.ibb.co/6HFswqX/900x300.gif`
We can block it like this? `||i.ibb.co/*/900x300.gif`
Second gif is occurs when the page is refreshed several times

Reply:
Yes, also add a modifier `$domain=spyhackerz.org` to those rules.